README.md

```diff
@@ -30,31 +30,9 @@ With [npm](https://npmjs.org) do:
 
 ## Methods
 
-### setConfig(configuration)
+### getStringResourceValue(componentName, key)
 
-Updates library configuration, which is an object containing the following keys:
-
-- `host`: the host where the server is running;
-- `port`: the port where the server is running;
-
-The following code snippet illustrates a call to the `setConfig(...)` method:
-
-```js
-xcfunctions.setConfig({
-    port: 8000,
-    host: 'localhost'
-});
-```
-
-### getConfig()
-
-Returns current library configuration.
-
-The following code snippet illustrates a call to the `getConfig()` method:
-
-```js
-xcfunctions.getConfig().port    // returns the port on which the server is running
-```
+Returns the string resource value of the specified component and key or null if any value wasn't found
```

Comment:
add example, be consistent to the structure  / documentation of the other functions

Reply:
done